src/bundle/ui-dev/src/modules/common/popup/popup.component.js

```diff
@@ -48,9 +49,21 @@ const Popup = ({
         if (isVisible) {
             showPopup();
             modalRef.current.addEventListener('hidden.bs.modal', onClose);
+        } else {
+            if (latestBootstrapModal.current) {
+                latestBootstrapModal.current.hide();
+            }
```

Comment:
up to you: :)
```suggestion
            latestBootstrapModal.current?.hide();
```
also it could be in elseif :)

Reply:
It was intended, I prefer `if`s ;P